src/Arcus.BackgroundJobs.Tests.Integration/appsettings.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "Arcus": {
+    "TenantId": "#{Arcus.KeyRotation.ServiceBus.TenantId}#",
```

Comment:
We'll have to change this to some more structure, but let us worry about that. Thx for already moving this to a more appropriate location.

Reply:
Working on this #110 . Will maybe have some failing tests here, we'll see. I'll try to be compatible.

Reply:
Since you're doing this via another PR, I guess we can resolve this conversation and close the PR @stijnmoreels  ?

Reply:
<img src="https://media4.giphy.com/media/l0ErOholJjSmFlMFG/giphy.gif"/>